docs/source/supplementary.rst

```diff
@@ -0,0 +1,1248 @@
+.. ...........................................................................
+.. © Copyright IBM Corporation 2020                                          .
+.. ...........................................................................
+
+Supplementary
+=============
+
+Encode
+------
+
+Many of the z/OS modules offer options to convert the data encodings that are
+read, copied, and transferred from a USS file or a data set. Encoding is managed
+by setting the source and destination character set options within the modules.
+
+A character set is an element of internationalization that maps and translates
+an alphabet or characters that are used in a particular language. A character
+set is made up of a series of code points or the numeric representation of a
+character. For example, the code point for the letter A in international EBCDIC
+is 0xC1.
```

Comment:
The definition is incomplete and im not in favor of using 'translating'. A
charset is not translating but just defines how an alphabet or character is
represented in a particular language. Please consider rewriting. IBM terminology
defines it differently
Suggested rewrite: A character set is an element of internationalization that
describes/ defines how an alphabet or character is represented in a particular
language.
IBM approved terminology describes it as :



noun
A defined set of characters with no coded representation assumed that can be recognized by a configured hardware or software system. A character set can be defined by alphabet, language, script, or any combination of these items.

Source: AIX Glossary, C/C++ for Workstations, C/C++ for z/OS, Content Manager OnDemand, DB2 Glossary, DB2 for z/OS, Dictionary of Printing, Globalization Leadership Team, IBM i, Language Environment, Rational Developer for IBM I, Terminology Extraction Process, Translation Services Center, z/OS UNIX System Services
Subject area: Information technology // Software
A set of binary codes that represent specific text characters.

Source: Domino, Notes, Terminology Extraction Process, Translation Services Center
Subject area: Information technology // Software
A list of characters (letters, numbers, and symbols such as #, $, and &) that are recognized by computer hardware and software.

Source: Sterling B2B Integrator
Subject area: Information technology // Software

source: http://tlwi.w3-969.ibm.com/standards/terminology/cgi-bin/lookup?ug=corporate&term=character+set&submit=Search&source=en&target_switch=none&template=simple&db_name=LOGOS&11=main+form&11=acronym%7Eabbreviation&11=prohibited

Reply:
I am ok with the suggested change, thank you.